### ISSUE TITLE
Update Wiki Content

### ISSUE THREAD

**Issue by goldbattle** [ethz-asl/kalibr#520]
- [ ] Download links https://github.com/ethz-asl/kalibr/wiki/downloads
- [ ] Install guide to user Docker https://github.com/ethz-asl/kalibr/wiki/installation
- [ ] Distortion models references https://github.com/ethz-asl/kalibr/wiki/Supported-models
- [ ] Packages to do allan variance https://github.com/ethz-asl/kalibr/wiki/IMU-Noise-Model
- [ ] Example wiki on calibrating T265 / realsense sensors
- [ ] Laser range finder guide https://github.com/ethz-asl/kalibr/wiki/Camera-IMU-LRF-calibration

**Comment by alvaro-oliver** [comment]
Would you explain in "Multi IMU and IMU intrinsic calibration" if is it possible to get intrinsic calibration values for a single IMU with no video reference?. What would be the experimental procedure for doing so?

A feature like that would expand the uses of Kalibr beyond inertial-video researchers, since Alberto Pretto's [IMU_Tk](https://bitbucket.org/alberto_pretto/imu_tk/src/master/) development is long time abandoned.

**Comment by goldbattle** [comment]
> Would you explain in "Multi IMU and IMU intrinsic calibration" if is it possible to get intrinsic calibration values for a single IMU with no video reference?. What would be the experimental procedure for doing so?
> 
> A feature like that would expand the uses of Kalibr beyond inertial-video researchers, since Alberto Pretto's [IMU_Tk](https://bitbucket.org/alberto_pretto/imu_tk/src/master/) development is long time abandoned.

I have updated the wiki's second paragraph to state that you need at least one camera.

**Comment by zoldaten** [comment]
a make a short post about Kalibr if you dont mind - https://habr.com/ru/post/689778/

**Comment by sgraybi** [comment]
@goldbattle maybe the attached document will help with "How to use IMU intrinsic calibration results".  Please verify for correctness!
[githubreply.md](https://github.com/ethz-asl/kalibr/files/11151500/githubreply.md)

**Comment by goldbattle** [comment]
> @goldbattle maybe the attached document will help with "How to use IMU intrinsic calibration results". Please verify for correctness! [KalibrHowToApplyCalibration.md](https://github.com/ethz-asl/kalibr/files/11151798/KalibrHowToApplyCalibration.md)

Great write up. Are you ok with me incorporating this into a wiki page for others to reference?

**Comment by sgraybi** [comment]
> > @goldbattle maybe the attached document will help with "How to use IMU intrinsic calibration results". Please verify for correctness! [KalibrHowToApplyCalibration.md](https://github.com/ethz-asl/kalibr/files/11151798/KalibrHowToApplyCalibration.md)
> 
> Great write up. Are you ok with me incorporating this into a wiki page for others to reference?

Yes, please do.  That was the intention.  Please verify the document for accuracy.  I believe I got the details correct but it would be great to get an expert review.

**Comment by sgraybi** [comment]
@goldbattle do you have any update on "How to use IMU intrinsic calibration results".  I have some accuracy issues in my system.  If I can rule out that I'm applying the calibration correctly that would help a lot!  Please feel free to link to other libraries or documentation if that is easier.
